docs/PROJECT_STRUCTURE.md

```diff
@@ -0,0 +1,74 @@
+# Project Structure
```

Comment:
We don't need this document - the project structure is pretty self explanatory (also the way this is defined is wrong, given we don't want to keep the existing structure in place)